commands/check.js

```diff
@@ -28,12 +28,28 @@ module.exports = {
     owner_only: false,
     affect_config: false,
     action: function(message, config) {
-        if (message.mentions.members.size == 0)
-            message.guild.fetchMember(message.author)
-            .then(guildMember => message.channel.send("", build_embed(guildMember, config)))
-        else
-            message.mentions.members.map(
-                guildMember => message.channel.send("", build_embed(guildMember, config))
+        mentioned_users = message.mentions.users
```

Comment:
You should have checked if my check command works for multiple mentions. (It does).